hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestBlockOutputStreamWithFailures.java

```diff
@@ -454,8 +463,8 @@ private void testExceptionDuringClose(OzoneClient client) throws Exception {
 
     assertInstanceOf(ContainerNotOpenException.class,
         checkForException(blockOutputStream.getIoException()));
-    assertTrue(checkForException(blockOutputStream
-        .getIoException()) instanceof ContainerNotOpenException);
+    assertInstanceOf(ContainerNotOpenException.class,
+        checkForException(blockOutputStream.getIoException()));
```

Comment:
This seems to be the same as the existing `assertInstanceOf` above.  I think we can keep one and remove the other.